ocaml/libs/sexpr/sExprLexer.mll

```diff
@@ -7,8 +7,6 @@ rule token = parse
   | [' ' '\t' '\r']                     { token lexbuf }
   | ';' [^ '\n']* '\n'                  { incr line; token lexbuf }
   | '\n'                                { incr line; token lexbuf }
```

Comment:
Not related to your change, but just spotted this one: incrementing line isn't how one is supposed to track positions in a lexer (well first of all I can't find any code that would read `line`). There is `Lexer.new_line` that updates it properly, and then error messages should contain an actual line:column position.

Although in practice this is used for deserializing fields from the xapi db, which are all single line.